README.md

```diff
@@ -27,71 +25,121 @@ Lance makes machine learning workflows with ML data easy (images, videos, point
 
 * Version, compare and diff ML datasets easily.
 
+* Search for nearest neighbors in under 1 millisecond.
+
 * (Coming soon) visualize, slice and drill-into datasets to inspect embeddings, labels/annotations and metrics.
 
 Lance is powered by Lance Format, an Apache-Arrow compatible columnar data format which is an alternative to Parquet, Iceberg and Delta. Lance has 50-100x faster query performance for ML data.
 
+
 ## Quick Start
 
+
+**Converting to Lance**
+
+```python
+import lance
```

Comment:
We have the import before installation, require `pip install pylance` above this